multicast/send.py

```
@@ -350,22 +392,39 @@ def doStep(self, *args, **kwargs):
 			# Read from stdin in chunks
 			while True:
 				try:
-					chunk = sys.stdin.read(1316)  # Read 1316 bytes at a time - matches read size
-				except IOError as e:
-					print(f"Error reading from stdin: {e}", file=sys.stderr)
+					# Read configured amount of bytes at a time - matches read size by default
+					# skipcq: PYL-W0212
+					chunk = sys.stdin.read(
+						multicast._MCAST_DEFAULT_BUFFER_SIZE,  # skipcq: PYL-W0212 - module ok
+					)
+				except IOError:
```

Ruff (0.8.2)

Replace aliased errors with `OSError`

Replace `IOError` with builtin `OSError`

(UP024)